### DISCUSSION TITLE
Overriding tailwind config to create custom styling?

### DISCUSSION THREAD

**Discussion by carlbarrdahl** [tailwindlabs/tailwindcss#13787] [Help]
I am working on a react component library and I want to allow the developers using the library to override tailwind settings with their own config? How can I accomplish this? 

Can my component library's tailwind config be overridden from a config outside the library?

**Comment by wongjn** [comment]
You could consider having consumers compile the CSS for your library, using [a Tailwind Preset](https://tailwindcss.com/docs/presets) provided by your library.

**Reply by carlbarrdahl** [reply]
Thank you @wongjn! This might be the way. Could you point me in a direction where I can learn about having consumers compile the CSS?

**Reply by wongjn** [reply]
That would be the consumers presumably using Tailwind to compile the CSS. You can find installation and integration guides [on the Tailwind website](https://tailwindcss.com/docs/installation).
